doc/developer/standards_and_practices.rst

```diff
@@ -299,12 +299,8 @@ zero pylint warnings or errors.
 General do's and don'ts
 =======================
 
-do not use ``print``
+Do not use ``print``.
     ARMI code should not use the ``print`` function; use one of the methods within ``armi.runLog``.
 
-do not use ``super``
```

Comment:
this is a pretty big difference in guidance. i thought we had good reason for it. it gets complex with multiple inheritance, which we also discourage. can you check with Nick? There are 2 ways to do it, and our code base is mostly limited to one way. it might be best to keep it that way

Reply:
I too was a bit surprised by this. I'm not as opposed to multiple inheritance (as long as it's done carefully), but am certainly curious as to what the motivation was behind the change. I would have expected a larger discussion amongst the ARMI devs for a change to the standards and practices.

Reply:
I can remove this from this PR and  make it it's own, if you like.

But I have never heard of a Python project with this rule. `super()` is just part of the Python language, and it makes long-term maintenance of OOP project significantly less tedious.

But, mostly, I am aware of no reason this rule would help us. And yesterday Chris K got bit by this problem. And frankly I think `super()` makes this PR cleaner.

Reply:
I'm not necessarily opposed to the change, I think it's reasonable; `super()` is standard python practice for inheritance, as I understand it. I was just hoping for a bit more dialogue on it - or any change to the standards and practices, really. In its current form, I think it would slide under the radar and most of the regular ARMI contributors would miss it.

I would advocate for pulling it out of the PR and opening an issue or discussion on it so that there is transparency amongst everyone. Just my 2 cents though!

Reply:
Well, the only reason I included it in this PR is because this PR makes a lot more sense with `super()`.  I can pull it out into it's own PR, and just wait on this one until the `super()` PR merges.

But here's the short version:  I do not see any benefit to the current opposition to `super()` use in our docs.  It seems like a "style points" rule made by someone with "strong opinions", and not a practical decision designed to help us out in any way.  After carefully researching it, I can't find any other Python open source projects using this rule. Because... why would they? What is the benefit of such a rule? 

Sure, we can open another ticket/discussion/PR.

Reply:
Okay, I moved this to another PR.